Dockerfile.generate

```diff
@@ -1,4 +1,4 @@
 FROM golang:1.9.2
 
 RUN apt-get update && \
-    apt-get install -y python-yaml
+    apt-get install -y python-yaml jq gawk
```

Comment:
`make generate` never actually worked :/
By default, Debian uses an alternative called `mawk`, which was not compatible with the GNU awk syntax in `hack/build-jsonnet.sh`. CI never caught this until now because the script was never actually triggered because the Make dependencies were funky.

Reply:
oops. nice catch!